byron/ledger/executable-spec/src/Ledger/UTxO.hs

```diff
@@ -68,11 +71,11 @@ fromTxOuts = UTxO . Map.fromList . fmap (\out -> (TxIn (mkId out) 0, out))
 data Tx = Tx
   { inputs  :: [TxIn]
   , outputs :: [TxOut]
-  } deriving (Eq, Show, Ord, Generic, Hashable)
+  } deriving (Eq, Show, Ord, Generic, Hashable, HasTypeReps)
 
-instance HasTypeReps Tx where
-  typeReps x@(Tx inputs outputs)
-    = typeOf x <| typeOf inputs <| typeOf outputs <| empty
+-- instance HasTypeReps Tx where
```

Comment:
Kill with :fire: ?

Reply:
I thought I'd keep it for writing the property tests and showing this version didn't work, but maybe that's a pointless test and we should just test that the new impl works!

Reply:
Oh, also fine with me :)